platform/riscv-virt/kernel_platform.h

```diff
@@ -1,8 +1,11 @@
 #define PHYSMEM_BASE          0x80000000
-#define BIOS_SIZE             (128*1024)
-#define INIT_PAGEMEM          (PHYSMEM_BASE+BIOS_SIZE)
+#define BIOS_SIZE             (128*KB)
```

Comment:
I have seen some newer OpenSBI images exceed 128KB. Not sure what a safe size is - maybe double this for now?

Reply:
Good idea, seems safer. I've updated it to 256K.